### PR TITLE
Disable properties prefix

### PR DESCRIPTION
**Pull request by luigibk**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/sscs-evidence-share/values.aat.template.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
+    PDF_SERVICE_BASE_URL: https://docmosis-development.platform.hmcts.net/rs/render
```
Comment (timja) [inline charts/sscs-evidence-share/values.aat.template.yaml]: shouldn't this be in values.yaml?

File: charts/sscs-evidence-share/values.preview.template.yaml [hunk 2]
```diff
@@ -4,22 +4,15 @@ java:
...
     SEND_LETTER_SERVICE_ENABLED: "true"
+    SEND_LETTER_SERVICE_BASEURL: "http://rpe-send-letter-service-aat.service.core-compute-aat.internal"
+    IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
+    CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
     SUBSCRIPTION_NAME: "sscs-evidenceshare-subscription-aat"
     TOPIC_NAME: "sscs-evidenceshare-topic-aat"
     AMQP_HOST: "sscs-servicebus-aat.servicebus.windows.net"
     AMQP_USERNAME: "SendAndListenSharedAccessKey"
-    PDF_SERVICE_BASE_URL: ${PDF_SERVICE_BASE_URL}rs/render
```
Comment (timja) [inline charts/sscs-evidence-share/values.preview.template.yaml]: ah, this was what made it work on preview...

File: charts/sscs-evidence-share/values.aat.template.yaml (hunk 1 above)
Comment (luigibk) [reply charts/sscs-evidence-share/values.aat.template.yaml]: Good idea. That way it only needs to be overridden in prod

File: charts/sscs-evidence-share/values.preview.template.yaml (hunk 2 above)
Comment (luigibk) [reply charts/sscs-evidence-share/values.preview.template.yaml]: Removing it from bootstrap.yaml really

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -54,7 +54,7 @@ send-letter:
 idam:
   url: ${IDAM_API_URL:http://localhost:5000}
   s2s-auth:
-    totp_secret: ${IDAM_S2S_AUTH_TOTP_SECRET:AAAAAAAAAAAAAAAC}
+    totp_secret: ${IDAM_S2S_AUTH_TOTP_SECRET:}
```
Comment (jackmaloney) [inline src/main/resources/application.yaml]: This is going to break local development

Comment (luigibk) [reply src/main/resources/application.yaml]: That should be ok to leave there. Let's try...

**Pull request merged**